x/crosschain/keeper/msg_server_migrate_tss_funds.go

```diff
@@ -181,6 +181,7 @@ func (k Keeper) MigrateTSSFundsForChain(
 				),
 			)
 		}
+		cctx.GetCurrentOutboundParam().GasPrice = multipliedGasPrice.String()
```

Comment:
We need to fill `GasPriorityFee` as well, do we? zetaclient will use it to transfer TSS funds thru a EIP-1559 transaction.

Reply:
Should we cover the case where the chain does not support 1559?

Reply:
It's there, check comments above. https://github.com/zeta-chain/node/pull/2483/files#r1679509456